### ISSUE TITLE
Theme colour for base styles 

### ISSUE THREAD

**Issue by paulrobertlloyd** [design-history/design-history#152]
I wonder if styles could include an additional option for further customisation.

For example, as a design history for a service within a UK government department, but not necessarily one for GOV.UK, I might want to use my department’s brand colour, for example DEFRA uses `#00a33b`, DLUHC `#012169`, Home Office `#9325b2`, etc. A setting that would set the SCSS `$govuk-brand-color` value would probably be enough.

Even within the NHS, while NHS England uses `#005eb8`, ~~NHSX~~ its Transformation Directorate uses a darker shade of blue: `#003087` (not sure if NHS Frontend has a similar SCSS variable). In fact, I note that https://scot.nhs.uk appears to be using that darker shade now as well (NHS Wales appears to use variations of cyan `#2d7ea5`).

**Comment by fofr** [comment]
The prototype has a _sort of_ draft of this feature:

<img width="762" alt="Screenshot 2023-01-23 at 17 45 39" src="https://user-images.githubusercontent.com/319055/214111641-da46cd81-f8c4-4f25-ac48-48189f06d32c.png">

**Comment by fofr** [comment]
For reference later:

Different department crests:
https://components.publishing.service.gov.uk/component-guide/organisation_logo
https://github.com/alphagov/govuk_publishing_components/blob/main/app/assets/stylesheets/govuk_publishing_components/components/_organisation-logo.scss

Different organisation colours:
https://github.com/alphagov/govuk-frontend/blob/main/src/govuk/settings/_colours-organisations.scss (this is available in govuk-frontend)

**Comment by paulrobertlloyd** [comment]
Yeah, bit of a rabbit hole this. I guess it depends how closely you want to tie the product to specific bits of UK (and devolved, regional and local) government, and how much customisation you wish to provide.

I wonder if there are a number of colour variables you can change, and then the current styles are presets on top of these (much like in Slack where you can pick a theme, but then go in and change certain values).

**Comment by paulrobertlloyd** [comment]
Just seen this, which appears to be extremely relevant: [How we let GOV.UK Notify users set their own branding](https://gds.blog.gov.uk/2023/01/23/how-we-let-gov-uk-notify-users-set-their-own-branding). Looking at the screenshots however… isn’t that going to be an extremely long list!?
